src/components/radio-group/radio-button.st.css

```diff
@@ -98,4 +98,3 @@
     margin-right: 10px;
 }
 
-
```

Comment:
result of branching, cant make this re-appear to remove the diff as the linter seems to automatically remove it